pytest.ini

```diff
@@ -9,6 +9,8 @@ markers =
     bucket_encryption
     cloud_transition
     encryption
+    encryption_sse_c
+    encryption_sse_kms
```

Comment:
there's already a marker for `sse_s3`. please remove the `encryption_` part to be consistent